modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/home/components/ProjectsList/ProjectList.test.js

```diff
@@ -0,0 +1,114 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ */
+
+import {faker} from '@faker-js/faker';
+import {render, screen} from '@testing-library/react';
+import ProjectList from '.';
+
+describe('render Project List component', () => {
+	const koroneikiAccounts = {
+		items: [
+			{
+				accountKey: `KOR-${faker.random.numeric(5)}`,
+				code: faker.commerce.product().toUpperCase(),
+				name: faker.commerce.productName(),
+				region: faker.address.country(),
+				slaCurrentEndDate: faker.date.future(),
+				status: 'Active',
```

Comment:
Put fixed values, I know it's an array, you can create a util for that, or reuse the variables, but if possible use fixed values is better

I imagine some properties inside this array should be unique, you can randomize if needed 

```suggestion
				accountKey: 'KOR-12345',
				code: 'COD-12345',
				name: 'PROD-12345',
				region: 'EUA',
				slaCurrentEndDate: '2022-06-17',
				status: 'Active',
```

Reply:
Done!